core/integration/module_test.go

```diff
@@ -3841,7 +3841,10 @@ import (
 type Test struct {}
 
 func (t *Test) Call(
-  //+ignore=["foo.txt", "bar"]
+  // +ignore=[
+  //   "foo.txt",
+  //  "bar"
```

Comment:
Not important, just stands out to me 🙂 
```suggestion
  //   "bar"
```